model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -159,4 +164,94 @@ class OrtResultTest : WordSpec({
+                            license = null,
+                            licenseSource = null,
+                            severity = Severity.HINT,
+                            message = "Rule violation message",
+                            howToFix = ""
+                        )
+                    )
+                )
+            )
+
+            val violations = ortResult.getRuleViolations()
+
+            violations should haveSize(1)
+        }
+
+        "drop resolved rule violations if omitResolved is true" {
+            val ortResult = OrtResult.EMPTY.copy(
+                repository = Repository.EMPTY.copy(
+                    config = RepositoryConfiguration(
+                        resolutions = Resolutions(
+                            ruleViolations = listOf(
+                                RuleViolationResolution(
+                                    "Rule violation message",
+                                    RuleViolationResolutionReason.EXAMPLE_OF_EXCEPTION,
+                                    "comment"
+                                )
+                            )
+                        )
+                    )
+                ),
+                evaluator = EvaluatorRun(
+                    violations = listOf(
+                        RuleViolation(
+                            rule = "rule id",
+                            pkg = Identifier("Maven", "org.ossreviewtoolkit", "resolved-violation", "0.8.15"),
+                            license = null,
+                            licenseSource = null,
+                            severity = Severity.ERROR,
+                            message = "Rule violation message",
+                            howToFix = ""
+                        ),
+                        RuleViolation(
+                            rule = "another rule",
+                            pkg = Identifier("Maven", "com.example", "package-without-resolution", "1.0.0"),
+                            license = null,
+                            licenseSource = null,
+                            severity = Severity.WARNING,
+                            message = "Message without any resolution",
+                            howToFix = ""
+                        ),
+                        RuleViolation(
+                            rule = "rule below threshold",
+                            pkg = Identifier("Maven", "com.example", "violation-below-threshold", "3.14"),
+                            license = null,
+                            licenseSource = null,
+                            severity = Severity.HINT,
+                            message = "Message without any resolution",
+                            howToFix = ""
+                        )
+                    )
+                )
+            )
+
+            val violations = ortResult.getRuleViolations(omitResolved = true, minSeverity = Severity.WARNING)
```

Comment:
The assertion could be made stronger with: 

```
ortResult.getRuleViolations(omitResolved = true, minSeverity = Severity.WARNING).map {it.rule } shouldContainExactlyInAnyOrder(
   "rule id",
   "another rule"
)
```
...same for the other introduced test further above.